src/components/cc-global-header.vue

```diff
@@ -124,11 +127,17 @@ export default defineComponent({
         { ID: 4, url: "#", title: "Menu 4" },
       ];
     } else {
+      vm.menuLoading = true;
       axios
         .get(requestUrl)
-        .then((response) => (vm.menus = response.data))
+        .then((response) => {
+          vm.menuLoading = false;
+          vm.errorMessage = null;
+          vm.menus = response.data;
+        })
         .catch((error) => {
-          vm.errorMessage = error.message;
+          vm.menuLoading = false;
+          vm.errorMessage = `We are having trouble loading the menu <br /> Error Message: ${error.message}.`;
```

Comment:
On second thought, let's keep HTML out of JavaScript since we have the `template` area for declarative markup.
```suggestion
          vm.errorMessage = error.message;
```